docs/pages/connect-your-client/introduction.mdx

```diff
@@ -250,6 +240,42 @@ cluster](../../img/use-teleport/kubernetes-login.png)
 </TabItem>
 </Tabs>
 
+### Transferring files
+
+<Tabs>
+<TabItem label="tsh scp">
+
+`tsh scp` will let you transfer files to servers behind Teleport:
+
+```code
+$ tsh scp some-file.ext server.example.com:
+some-file.ext   7% |███████                | (25/342 MB, 2.9 MB/s) [9s:1m48s]
+```
+</TabItem>
+<TabItem label="Teleport Connect">
+
+Teleport Connect allows you to transfer files to a remote server and runs on MacOS, Linux and Windows. You can use the arrows in the top-left corner of an active SSH session to upload and download files:
+
+![Download with connect](../../img/download_files_connect.png)
```

Comment:
We'll need to update the image reference here

Reply:
I have been saving these in the `img` directory and they're all resolving for me:

```
-rw-r--r--@  1 mmcallister  staff    18024 May 22 12:20 upload_files_webui.png
-rw-r--r--@  1 mmcallister  staff    12826 May 22 12:21 download_files_webui.png
-rw-r--r--@  1 mmcallister  staff    50658 May 23 11:07 download_files_connect.png
-rw-r--r--@  1 mmcallister  staff    59520 May 23 11:53 upload_files_connect_drag_drop.png
/Users/mmcallister/Teleport/teleport/docs/img [copy_files_guide]
```